ex_webrtc/integration_test/test_videoroom/lib/test_videoroom/application.ex

```diff
@@ -7,7 +7,13 @@ defmodule TestVideoroom.Application do
   def start(_type, _args) do
     children = [
       {Phoenix.PubSub, name: TestVideoroom.PubSub},
-      TestVideoroomWeb.Endpoint
+      TestVideoroomWeb.Endpoint,
+      {Membrane.TelemetryMetrics.Reporter,
+       [
+         metrics: Membrane.RTC.Engine.Endpoint.ExWebRTC.Metrics.metrics(),
+         name: ExWebrtcMetricsReporter
+       ]},
+      TestVideoroom.MetricsScraper
```

Comment:
What is the advantage of using Membrane.TelemetryMetrics instead of regular telemetry in case of ex_webrtc_endpoint ?

Reply:
The advantage of using Membrane.TelemetryMetrics over regular Telemetry.Metrics, especially in the context of ex_webrtc_endpoint, stems from the specific adaptations in Membrane.TelemetryMetrics that are tailored for multimedia applications. Felix mentioned that regular Telemetry.Metrics might not be sufficiently suited for multimedia use cases. Furthermore, Membrane.TelemetryMetrics includes certain optimizations or "hacks" that you would need to manually implement if you were using Telemetry.Metrics, enhancing efficiency and effectiveness for multimedia-specific metrics collection and monitoring in the Membrane framework.